src/javascript/app/pages/user/account/settings/financial_assessment.js

```diff
@@ -45,6 +45,11 @@ const FinancialAssessment = (() => {
             });
         }
 
+        const is_professional = State.getResponse('get_account_status.professional') || State.getResponse('get_account_status.professional_requested');
```

Comment:
what if we store `State.getResponse('get_account_status')` in one variable e.g. `const get_account_status` and then do `get_account_status.professional || get_account_status.professional_requested` to avoid calling state twice?

Reply:
ok, sure